RELEASE.md

```diff
@@ -0,0 +1,113 @@
+# web3.js Release Guidelines
+
+To provide the community with latest features and patches as soon as possible, we follow the `ship-it` philosophy along with [conventional commit messages](https://www.conventionalcommits.org/en/v1.0.0/).
+
+## Version Number Definition
+
+The web3.js project follows the [semver 2.0.0 specification](https://semver.org/).
+
+### Major
+
+The `major` version has to be increased as soon as a breaking change is introduced. The definition of a breaking change is anything that requires a depending project to update their code base, build pipeline or tests.
+
+### Minor
+
+The `minor` is increased as soon as new smaller features do get introduced. A `minor` release will not affect any depending project. Such a release only introduces smaller enhancements and features a project could use.
+
+### Patch
+
+A patch release only contains required bug fixes with a low risk to impact depending project.
+
+Further details about versioning can be found in the [semver 2.0.0 specification](https://semver.org/) we follow.
+
+## Release Process Rules
+
+### Major
+
+1.  The release should get released as an `RC` version on the specified track below.
+1.  The minor `RC` version stays released for a `month` for testing purposes.
+1.  During release review, the code is frozen unless new changes are proposed, approved and merged.
+    1.  Changes should trigger a new `RC` release and set the release clock back a `week` so reviewers have the time they need to test new changes.
+1.  The project lead should manually test the `RC` against a Node project and the published
+    minified bundle in a browser context. An external reviewer should verify they have done the same.
+
+### Minor
+
+The `minor` release inherits the rules from the `major` release but has a smaller testing phase of one `week` and the release clock will be set back `2 days` on a change of the published code.
+
+### Patch
+
+Patch releases do not get any further rules and can be released on a `nightly` basis.
```

Comment:
Since November, Web3 has been following the `Minor` rules for `Patch` as well, to help establish a series of non-breaking releases.  This line would be a change from existing practice...

Reply:
thanks, will adjust accordingly